### PR TITLE
Add support for Node Corepack

### PR DESCRIPTION
**Pull request by wojtekmaj**
Node ships with Corepack, a "package manager manager". It works by reading `packageManager` entry in `package.json` and downloading required binaries transparently, on-the-fly. Therefore it ensures that all developers using Corepack run the same version of package manager:

- reducing changes for any incompatibilities that might be caused by different versions,
- making installation seamless - Yarn is installed transparently without any intervention, provided you have Corepack enabled
- making migration to e.g. modern Yarn or pnpm very easy, as developers don't need to take any further actions besides pulling the latest changes from origin

This PR does not break the repository for those using Yarn Classic (v1) installed using standalone installer, while adding an easy way to contribute for those who did not install it.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,446 +1,51 @@
...
-
-### Bug Fixes
-
-* no more using SFC types ([ca9e5e8](https://github.com/theKashey/react-focus-lock/commit/ca9e5e8ae1bbb4ee252a11490f95c2dcc2f76efd))
-* update dependnecies. fixes [#210](https://github.com/theKashey/react-focus-lock/issues/210) and fixes [#212](https://github.com/theKashey/react-focus-lock/issues/212) ([a3895ce](https://github.com/theKashey/react-focus-lock/commit/a3895cec02b2a748f85424c2dabaf9c8b698b76c))
-
-
-
-## [2.8.1](https://github.com/theKashey/react-focus-lock/compare/v2.8.0...v2.8.1) (2022-02-14)
+## 2.9.5 (2023-10-11)
```
Comment (theKashey) [inline CHANGELOG.md:1]: can you please stash changes in this file 🙏

File: package.json [hunk 2]
```diff
@@ -106,5 +106,6 @@
     "react-clientside-effect": "^1.2.6",
     "use-callback-ref": "^1.3.0",
     "use-sidecar": "^1.1.2"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```
Comment (theKashey) [inline package.json:110]: 👍

**Review by theKashey — APPROVED** [review]

**Comment by wojtekmaj** [comment]
Sure, done!

File: CHANGELOG.md (hunk 1 above)
Comment (wojtekmaj) [reply CHANGELOG.md]: Sure, done!

**Pull request merged**